test/integration/basic/pages/throw-undefined.js

```diff
@@ -0,0 +1,9 @@
+export default function ThrowUndefined (props) {
+  return (
+    <div>throw-undefined</div>
+  )
+}
+
+ThrowUndefined.getInitialProps = () => {
```

Comment:
We should export at the bottom after this

Reply:
Updated